### PR TITLE
[Merged by Bors] - Document Homebrew package

### LINKED ISSUE #2329: Apple Silicon M1 support?
I would like to know if Lighthouse supports Apple Silicon M1 chip?

### PR DESCRIPTION
**Pull request by riptl**
## Issue Addressed

Resolves #2329

## Proposed Changes

Documents the recently added `lighthouse` Homebrew formula.

## Additional Info

NA

### CODE REVIEW

**Comment by divagant-martian** [comment]
Thanks for your contribution @terorie! 

Right now, every installation mode described in the book (binaries for various platforms + docker + source) are maintained by the lighthouse team. Under the same strategy, I'd argue in favor of moving the `Homebrew` chapter to `Community maintained installers` with an entry for homebrew. It would be good too adding a direct link to the formula's page so that users can check the target platform for this installer directly where it is documented.

Let me know what you think

**Comment by riptl** [comment]
> Under the same strategy, I'd argue in favor of moving the Homebrew chapter to Community maintained installers with an entry for homebrew.

Makes sense, will do 👍🏻 

> It would be good too adding a direct link to the formula's page so that users can check the target platform for this installer directly where it is documented.

The PR already has links to https://formulae.brew.sh/formula/lighthouse and the source, is this okay?

**Review by michaelsproul — APPROVED** [review]
Thanks!

**Review by michaelsproul — CHANGES_REQUESTED** [review]
Ah, could you please rebase on `unstable` and change the target branch to `unstable`? :pray:

**Comment by riptl** [comment]
@michaelsproul Yes, sir! Done

File: book/src/homebrew.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+Alternatively, you can find the `lighthouse` binary at:
+
+```bash
+"$(brew --prefix)/bin/lighthouse" --help
+```
+
+### Maintenance
+
+The [formula] is kept up-to-date by the Homebrew community and a bot that lists for new releases.
```
Comment (michaelsproul) [inline book/src/homebrew.md]:
```suggestion
The [formula][] is kept up-to-date by the Homebrew community and a bot that lists for new releases.
```

No-op change to trigger Github's CI

**Review by michaelsproul — APPROVED** [review]

**Pull request closed without merging**